docs/index.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+title: Home
+nav_order: 1
+description: "Spanner migration tool is a stand-alone open source tool for Cloud Spanner evaluation and migration."
+permalink: /
+---
+
+# Spanner migration tool
+{: .fs-9 }
+
+Spanner migration tool (SMT) is a stand-alone open source tool for Cloud Spanner evaluation and migration.
+{: .fs-6 .fw-300 }
+
+[Learn More](#what-does-spanner-migration-tool-do){: .btn .btn-primary .fs-5 .mb-4 .mb-md-0 .mr-2 }
+[View it on GitHub][SMT repo]{: .btn .fs-5 .mb-4 .mb-md-0 }
+
+---
+
+Spanner migration tool is a stand-alone open source tool for Cloud Spanner evaluation and
+migration, using data from an existing PostgreSQL, MySQL, SQL Server, Oracle or DynamoDB database.
+The tool ingests schema and data from either a pg_dump/mysqldump file or directly
+from the source database, and supports both schema and data migration.
+
+## What does Spanner migration tool do?
+
+### Schema migrations
+
+Spanner migration tool is designed to simplify Spanner evaluation and migration. It automatically builds a Spanner schema from the schema of the source database. This schema can be customized using the web based schema assistant UI. Expect that you'll need to tweak and enhance what Spanner migration tool produces.
+
+### Data migrations
+
+Spanner migration tool supports both small-scale on-prem POC migrations as well as production grade minimal downtime migrations using GCP services (**Datastream and Dataflow**) -
```

Comment:
Probably have the support for production grade before mentioning the POC use case? When I read it, it gave a slight impression that POC is the major use case. You can chose to ignore this comment as well.

Reply:
Great feedback! Updated.